tests/src/CoreMangLib/system/span/BasicSpanTest.cs

```diff
@@ -637,6 +717,45 @@ static void CanCreateSpanFromString()
         AssertEqualContent(secondHalfOfString, spanFromSecondHalf);
     }
 
+    static void WhenStartLargerThanLengthThrowsExceptionStartCtor()
```

Comment:
This is not a proper XUnit test. This test is a stopgap measure to get some testing going for fast Span until it gets to  corefx.